frontend/ui/detailview.jsx

```diff
@@ -420,6 +428,36 @@ class DetailViewComponent extends React.Component {
   }
 
   render() {
+    const legends = this.getLegend().sort((a, b) => a.title < b.title);
+    const {
+      measure,
+      platform,
+      timeInterval
+    } = this.state;
```

Comment:
I would prefer not to do this sort of thing unless you're using the variables in question a fair amount (i.e. at least more than once). It makes it easier to understand where the variable came from if you address it by `this.state`.